packages/stacked_services/lib/src/snackbar/snackbar_service.dart

```diff
@@ -68,7 +68,7 @@ class SnackbarService {
     Get.snackbar(
       title,
       message,
-      titleText: _snackbarConfig?.titleColor != null || title.isNotEmpty
```

Comment:
I don't see a reason why a `_snackbarConfig?.titleColor` should play a role whether the title widget is visible, so I deleted those conditions. Corrent me if I'm missing something